src/compton/Compton.cc

```diff
@@ -107,25 +123,47 @@ Compton::Compton(const std::string &filehandle,
  * \param etemp The SCALED electron temperature ( temp / electron rest-mass )
  * \return      n_grp x n_grp x n_xi interpolated scattering kernel values
  */
-std::vector<std::vector<std::vector<double>>>
-Compton::interpolate(const double etemp) {
+std::vector<std::vector<std::vector<std::vector<double>>>>
+Compton::interpolate_csk(const double etemp) {
```

Comment:
Okay! I'd like to talk with you about the best way to 'flatten' this data before I make any changes :)